tests/Test.ADAL.NET.Unit/UnitTests.cs

```diff
@@ -281,7 +281,33 @@ public void SignWithCertificateTest()
             }
         }
 
-        [TestMethod]
+	    [TestMethod]
+	    [Description("Test to verify AdditionalClaims are part of the signed message in the ClientAssertion")]
+	    public void AdditionalClaimsSignWithCertificateTest()
+	    {
+			var x509Certificate = new X509Certificate2("valid_cert.pfx", TestConstants.DefaultPassword);
+
+		    const string clientId = "911";
+		    const string audience = "007";
+			const string clientipKey = "client_ip";
+		    const string clientipValue = "192.168.1.0";
+
+		    var additionalClaims = new Dictionary<string, string> {{clientipKey, clientipValue}};
+		    var clientCertificate = new ClientAssertionCertificate(clientId, x509Certificate, additionalClaims);
+		    var jsonWebToken = new JsonWebToken(clientCertificate, audience);
+		    var ipClaimAssertion = jsonWebToken.Sign(clientCertificate);
+
+		    Assert.AreEqual(clientId, ipClaimAssertion.ClientId);
+			Assert.IsNotNull(ipClaimAssertion.Assertion);
```

Comment:
indent issue (ident line 307)

Reply:
strange. do we use tab or space in the project? perhaps my vs settings are messed up.

Reply:
spaces 🛰

Reply:
sorry. my vs settings are messed up. (prev. org. insisted on tab's)

---
In reply to: [161831318](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/942#discussion_r161831318) [](ancestors = 161831318)